lib/nodejs/parallel-buffered-runner.js

```diff
@@ -14,6 +14,18 @@ const {BufferedWorkerPool} = require('./buffered-worker-pool');
 const {setInterval, clearInterval} = global;
 const {createMap} = require('../utils');
 
+/**
```

Comment:
certain options are not supported by worker processes, listed below.  first and foremost, a worker cannot run mocha in parallel 😄 

these options are stripped out before `Mocha` is instantiated.

`globalSetup`/`globalTeardown` are the responsibility of the _main process only_, by design